src/containers/NewsSearch.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { getArticles, searchArticle } from '../fetches/articleFetches.js';
+import ArticleList from '../components/articles/ArticleList';
+import Search from '../components/articles/Search';
+
+export default class NewsSearch extends Component {
+    state = {
+        loading: false,
+        articles: [],
+        search: ''
+
+    }
+
+    componentDidMount = async () => {
+        this.setState({ loading: true });
+        const articles = await getArticles();
+        await this.setState({ articles: articles.articles })
+        this.setState({ loading: false });
+    };
+    handleStateChange = (e) => {
+        this.setState({ search: e.target.value });
+        console.log(this.state.search);
+    }
+    handleSearchState = async (e) => {
+        e.preventDefault();
+        this.setState({ loading: true })
+        const articles = await searchArticle(this.state.search);
```

Comment:
I noticed in your fetch you to do json => json.result. The shape of the data has an array called "articles", you could destructor {articles} on like 27...but that may mean refactoring your components. Try to hit a route in postman first to see the shape of the data and plan accordingly.